examples/resources/configuration/subscriptions.yaml

```diff
@@ -0,0 +1,137 @@
+      version: v1alpha1
+    name: Subscriptions
+    category: Configuration
+    scope: namespace
+    description: '{{[Subscription](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/evnt-01-subscription#documentation-content)}} is used to subscribe to events.'
+    urlPath: subscriptions
+  details: |-
+    header: 
+      - name: status.conditions.status
+        source: 'status.ready = false ? "error" : "ready"'
+        widget: Badge
+        description: status.conditions.message
+      - name: spec.sink
+        source: spec.sink
+      - source: spec.sink
+        widget: ResourceLink
+        name: spec.sink.controlledBy
+        linkText: "'spec.sink.link'"
+        resource:
+          name: '$substringBefore($substringAfter(data, "http://"), ".")'
+          namespace: root.metadata.namespace
+          kind: "'Service'"
+    body: 
+      - source: status.conditions
+        widget: Table
+        name: status.conditions
+        children:
+          - source: 'status.ready = false ? "error" : "ready"'
+            name: status.type
+            widget: Badge
+            description: status.conditions.message
+          - source: '$item.status'
+            name: status.conditions.status
+          - source: '$item.reason'
+            name: status.conditions.reason
+          - source: '$readableTimestamp($item.lastTransitionTime)'
+            name: status.conditions.lastTransitionTime
+      - source: spec.filter.filters
+        widget: Table
+        name: spec.filters
+        disablePadding: true
+        showHeader: false
+        collapsibleTitle: '"Filter #" & $string($index + 1) & " " & ($item.eventType.value)'
+        collapsible:
+          - widget: Columns
+            children:
+              - source: '$item.eventType'
+                widget: Panel
+                name: spec.filter.eventType
+                children:
+                  - source: '$parent.property'
+                    name: spec.filter.event.property
+                  - source: '$parent.type'
+                    name: spec.filter.event.type
+                  - source: '$parent.value'
+                    name: spec.filter.event.value
+              - source: '$item.eventSource'
+                widget: Panel
+                name: spec.filter.eventSource
+                children:
+                  - source: '$parent.property'
+                    name: spec.filter.event.property
+                  - source: '$parent.type'
+                    name: spec.filter.event.type
+                  - source: '$parent.value = "" ? "Handled by the NATS backend" : $parent.value'
+                    name: spec.filter.event.value
+      - widget: EventList
+        filter: '$matchEvents($item, $root.kind, $root.metadata.name)'
+        name: events
+        defaultType: NORMAL
+        hideInvolvedObjects: true
+  form: |-
+    - path: spec.sink
+      name: spec.sink
+      required: true
```

Comment:
I don't think that `required` flag is needed here. It's automatically set based on the schema.